impeller/entity/shaders/texture_fill_external.frag

```diff
@@ -10,7 +10,9 @@ in float v_alpha;
 out vec4 frag_color;
 
 void main() {
-  vec4 sampled =
-      texture(SAMPLER_EXTERNAL_OES_texture_sampler, v_texture_coords);
+  vec2 tc = v_texture_coords;
```

Comment:
Nit: Add todo.

Filed an issue for follow-up investigation: https://github.com/flutter/flutter/issues/132360

```suggestion
  // TODO(132360): For some reason, the vertex shader Y invert isn't working
  //               even when setting the wrapped texture intent to
  //               `TextureIntent::kRenderToTexture` for external GL textures.
  //               In the meantime, performing the sampling invert here is safe
  //               because the Android embedder only tries to composite external
  //               textures that have been written to as a render target.
  vec2 tc = v_texture_coords;
```

Reply:
This is no longer necessary after setting the texture intent to be kUploadFromHost.

Reply:
Also note that the vertex shader is no longer modified.